Doc/library/turtle.rst

```diff
@@ -213,6 +213,32 @@ useful when working with learners for whom typing is not a skill.
     use turtle graphics with a learner.
 
 
+Automatically begin and end filling
+-----------------------------------
+
+If you have Python 3.14 or later, you don't need to call :func:`begin_fill` and
+:func:`end_fill` for filling. Instead, you can use the :func:`fill`
+:term:`context manager` to automatically begin and end fill. Here is an
+example::
+
+   with fill():
+      for i in range(4):
+         forward(100)
+         right(90)
+
+   forward(200)
+
+The code above is equivalent to::
+
+   begin_fill()
+   for i in range(4):
+      forward(100)
+      right(90)
+   end_fill()
+
+   forward(200)
```

Comment:
Perhaps it can be consistent with the following content
```
      >>> turtle.color("black", "red")
      >>> with turtle.fill():
      ...     turtle.circle(80)
```
(I mean **`>>>`**)

Reply:
It's not particularly useful if there's no output.

Reply:
It's not particularly useful if there's no output.